crates/bevy_scene/src/serde.rs

```diff
@@ -318,55 +349,55 @@ impl<'a, 'de> Visitor<'de> for SceneEntityVisitor<'a> {
     }
 }
 
-pub struct ComponentDeserializer<'a> {
+pub struct ReflectDeserializer<'a> {
```

Comment:
This could be a little confusing— especially since it's `pub`. What about `ReflectMapDeserializer` or something?

Naming things is hard haha 😅